net.mancubus.SLADE.yaml

```diff
@@ -0,0 +1,137 @@
+app-id: net.mancubus.SLADE
+runtime: org.freedesktop.Platform
+runtime-version: '18.08'
+sdk: org.freedesktop.Sdk
+command: slade
+finish-args:
+- --device=dri
+- --socket=x11
+- --share=ipc
+- --socket=pulseaudio
+- --env=DOOMWADDIR=/app/share/games/doom
+- --persist=.config/gzdoom
+- --filesystem=home
+cleanup:
+- /app/include
+- /app/lib/*.a
+- /app/lib/*.la
+- /app/lib/pkgconfig
+modules:
+
+- name: webkitgtk
```

Comment:
Isn't it already part of GNOME platform?

Reply:
True, I've switched to the Gnome SDK.